modules/installation-user-infra-machines-static-network.adoc

```diff
@@ -30,6 +30,9 @@ Ordering is important when adding kernel arguments: `ip=`, `nameserver=`, and th
 ====
 
 .Routing and bonding options for ISO
+
+The table below provides examples for configuring networking of your {op-system-first} nodes.  These are networking options that are passed to `dracut` during the boot of the system.  For more information about the networking options supported by `dracut`, please see the manpage for `dracut.cmdline`.
```

Comment:
```suggestion
The following table provides examples for configuring networking of your {op-system-first} nodes. These are networking options that are passed to the `dracut` tool during system boot. For more information about the networking options supported by `dracut`, see the `dracut.cmdline` manual page.
```
These are suggestions based on IBM Style Guide suggestions and interestingly a recent docs team conversation about `man page` vs. `manual page` sparked by `kdump` work from @kelvinfan001. :)